backend/backend-schedule/src/main/java/eu/socialedge/hermes/backend/schedule/domain/Trip.java

```diff
@@ -90,4 +90,18 @@ public void removeStop(Stop stop) {
     public List<Stop> getStops() {
         return Collections.unmodifiableList(stops);
     }
+
+    /**
+     * Returns the last stop's name as a headsign if give one is blank
+     * @param headsign head sign to test
+     * @param stops trip stops
+     * @return not blank headsign
+     */
+    public static String guessHeadsignIfBlank(String headsign, List<Stop> stops) {
```

Comment:
It seems to me that this method should not care about checking on 'headsigh' parameter blank or something. Method's purpose is to contain logic that evaluates headsign from the list of stops - let's not mix it with something else